ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -58,6 +61,8 @@ class FRONTEND_API FrontEndManager final {
         return load_by_model_impl({make_variant(vars)...});
     }
 
+    FrontEnd::Ptr load_by_model_impl(const std::vector<std::shared_ptr<Variant>>& variants);
```

Comment:
why template implementation above does not work?

Reply:
because you dont need to wrap std::vector<std::shared_ptr<Variant>> with variant as it is already vector of variants.

Reply:
Moving `**_impl` to public API is not good approach, needs to be solved somehow different

Reply:
sure, will try to specialize load_by_model_impl for std::vector<std::shared_ptr<Variant>>

Reply:
done.